### DISCUSSION TITLE
Welcome to woonuxt Discussions!

### DISCUSSION THREAD

**Discussion by scottyzen** [scottyzen/woonuxt#110] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by opestro** [comment]
Hi , im using woonuxt but i did some changes on it , im using Rest api instead of graphql for woocommerce with express js for security and to hide api , 
The question is why i moved from graphql to rest api !
The answer is easy , because woographql has less params and fetching features  than rest api !
I have a store with 4k products , it was so hard to handl all this products with gql .
So if you want to see the changes i can send you the update with backend ( express js )

**Reply by scottyzen** [reply]
Sure, send me the repo. I'd love to take a look and see the pros and cons. Thanks

**Comment by fabdao** [comment]
Hi everybody,

I was looking for long time a headless solution with WooCommerce and Vue JS. This repo answer all my need. 

I share in [Discussions](https://github.com/scottyzen/woonuxt/discussions/317)  a repo with WooNuxt running inside Docker if anyone could be interested. 

Best regard.
